application/src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/GbfsFreeVehicleStatusMapper.java

```diff
@@ -85,6 +86,10 @@ public VehicleRentalVehicle mapFreeVehicleStatus(GBFSBike vehicle) {
         return null;
       }
       rentalVehicle.fuel = new RentalVehicleFuel(fuelRatio, rangeMeters);
+      String availableUntil = vehicle.getAvailableUntil();
+      if (availableUntil != null && !availableUntil.isBlank()) {
```

Comment:
I think we have something in `StringUtils` for that. PLease check.

Reply:
You're right. 
I changed it. 
I used the method of the OTP `StringUtils`, not the method of Apache. I hope, that's the right way.

Reply:
Correct. Our own version.